workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsTabContent.tsx

```diff
@@ -35,11 +35,7 @@ export const WorkflowDetailsTabContent = ({
   loading,
   workflowOverviewDTO,
 }: Props) => {
-  const { workflowId, format } = useRouteRefParams(workflowRouteRef);
-  const workflowFormat = useMemo(
-    () => (format === 'json' ? 'json' : 'yaml'),
-    [format],
-  );
+  const { workflowId } = useRouteRefParams(workflowRouteRef);
```

Comment:
small improvement suggestion:
we can also take the workflow id from workflowOverviewDTO
this way the component becomes completely independent from the routing system, making it more reusable and isolated
WDYT?

Reply:
good idea